package.json

```diff
@@ -23,31 +23,32 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
     "ember-ajax": "2.0.1",
-    "ember-cli": "2.4.3",
+    "ember-cli": "2.7.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-qunit": "^1.4.0",
-    "ember-cli-release": "0.2.8",
+    "ember-cli-jshint": "^1.0.0",
+    "ember-cli-qunit": "^2.0.0",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-qunit": "^2.1.0",
+    "ember-cli-release": "0.2.9",
     "ember-cli-sri": "^2.1.0",
+    "ember-cli-test-loader": "^1.1.0",
+    "ember-data": "^2.7.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
-    "ember-hash-helper-polyfill": "^0.1.1",
```

Comment:
Ember says it is no longer necessary

<img width="875" alt="screen shot 2016-07-28 at 11 24 38 am" src="https://cloud.githubusercontent.com/assets/2779962/17224474/e592d4d4-54b5-11e6-8968-2c944126745e.png">